src/components/contracts/ContractsDialog.js

```diff
@@ -74,6 +74,8 @@ const ContractsDialog = ({
   const dispatch = useDispatch();
   const queryClient = useQueryClient();
   const hasSubContractEnabled = !!contractFields.find((c) => c.code == "contract_main_orgunit");
+  const isContractViewPage =
```

Comment:
this looks like a hack, I would pass a boolean prop in the <ContractDialog isContractViewPage={true} /> when used in the ContractPage